### DISCUSSION TITLE
"Inter" vs "Inter var" font for TailwindUI

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#2151] [General]
https://github.com/rsms/inter
I see that Inter var ships only woff2 format.

Is there any difference when using them with TailwindUI?
What is the recommended approach?

**Comment by harveyhq** [comment]
Inter is offered as both traditional constant font files (one per style, e.g. Bold Italic, Medium, etc.) as well as a [Variable Font](https://en.wikipedia.org/wiki/Variable_fonts) which contains all styles in a much smaller file size. 👍
